ansible/roles/blackbox_exporter/templates/blackbox_exporter.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Prometheus Blackbox Exporter
+
+[Service]
+User=nobody
+Group=nogroup
+EnvironmentFile=-/etc/default/blackbox_exporter
+PermissionsStartOnly=true
+ExecStartPre=/sbin/setcap 'cap_net_raw+ep' {{ blackbox_exporter_path }}/blackbox_exporter
```

Comment:
It's only needed for ICMP.  DNS is done through normal resolver methods.

The actual implementation is based on the upstream Go code: https://godoc.org/golang.org/x/net/icmp

It looks like this used to not work, but should work now: https://github.com/golang/go/issues/9166

We should test this with minimal privs. If it works, I can update the upstream docs.